nixos/tests/all-tests.nix

```diff
@@ -19,6 +19,9 @@ let
   handleTestOn = systems: path: args:
     if elem system systems then handleTest path args
     else {};
+
+  nixosLib = import ../lib {};
+  evalModule = module: nixosLib.evalModules { modules = [ module ]; };
```

Comment:
Oh boy, don't we have a better name? It's the whole `option` vs `options` all over again

Reply:
This being test code, it isn't quite as critical, but runs the risk of becoming convention. I've changed it to `evalMinimalConfig`.